### DISCUSSION TITLE
Quasar deployed on Vercel cloud functions

### DISCUSSION THREAD

**Discussion by franco-onevillas** [vercel/community#319] [Help] [answered]
Hi there,

I am currently working with Quasar (vue) framework and I am trying to deploy it on serverless on Vercel as stated here: https://quasar.dev/quasar-cli/developing-ssr/ssr-production-export#serverless

But when I try to test it on local, run `vercel dev` and go to `/api` the console logs the things that usually logs on normal SSR but I never get the resource on the browser due to a timed out.

<img width="564" alt="image" src="https://user-images.githubusercontent.com/92261640/152704950-b47487ca-add5-4b5c-bece-c58dadbef580.png">

Any clue?

Thanks in advance

**Comment by ismaelrumzan** [comment, marked as answer]
Hi there,
Please review the following link that explains this error in more detail:
https://vercel.com/docs/concepts/functions/serverless-functions#execution-timeout
I would suggest checking your function related to this checklist:
https://vercel.com/docs/concepts/functions/serverless-functions#timeout-conditions-checklist
Have you tested your code by running it locally?

**Reply by franco-onevillas** [reply]
Hi Ismael, thank you very much for your answer.

I tried this things :) and tbh, I ended up moving to firebase which works smoothly. Probably some integration thing...

I can't close this!

Thank you again.

**Reply by ismaelrumzan** [reply]
Thanks for letting us know. I'll mark this as the answer

**Reply by franco-onevillas** [reply]
Thank you Ismael, let me know if you need a tester in case you expand to Quasar!
